test/spspSpec.js

```diff
@@ -118,11 +118,11 @@ describe('SPSP Module', function () {
         .get('/spsp')
         .reply(200, spspResponse)
 
-      const payment = yield SPSP.quoteDestination(this.plugin, 'alice@example.com', '10')
+      const payment = yield SPSP.quoteSource(this.plugin, 'alice@example.com', '12')
       assert.deepEqual(payment, {
         destinationAccount: "example.alice",
         connectorAccount: "example.connie",
-        sourceAmount: "10",
+        sourceAmount: "12",
```

Comment:
Minor comment but just from the perspective of the readability of the tests, I think it would make more sense if the source and destination amounts were the same in both the source and destination quoting cases